project_issue_reassign/README.rst

```diff
@@ -0,0 +1,38 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+The Responsible (``user_id``) field is made read only, and can instead be
```

Comment:
OK then. I don't remember problem with a doubled field with groups, because it's filtered of _fields_view_get_.